src/components/Form/Schemas/index.js

```diff
@@ -173,7 +173,7 @@ export default class Schemas extends React.Component {
 
     getSchemas(schemas) {
         return schemas
-            .filter(i => i.metadata)
+            .filter(i => i.metadata && !EXCLUDE_SCHEMAS.has(i.collectionName))
             .map(i => ({
```

Comment:
This is nice and readable, but slow. If this is executed often, perhaps replace with a single `reduce`

Reply:
It's not slower than before. And Schemas has at most like 90 elements. Also why I changed EXCLUDE_SCHEMAS to be a set instead of an array, so we dont need a O(n) lookup for every element. 
It's also only run once (per metadata-export page visit); when the Schemas component mounts. I will actually look into caching the schemas, as its unecessary to load this every time you visit the page.